frontend/src/components/utils/Search.jsx

```diff
@@ -1,141 +1,128 @@
-              >
+  render() {
+    return (
+      <form onSubmit={this.handleSearch} className="search-box">
+        <Row noGutters>
+          <InputGroup>
+            {!this.props.dashboard && (
+              <InputGroup.Prepend>
+                <InputGroup.Text id="search">
+                  <img
+                    src="/static/images/icon-search.svg"
+                    className="search-icon"
+                  />
+                </InputGroup.Text>
+              </InputGroup.Prepend>
+            )}
+            <FormControl
+              placeholder="Search for Account ID, Txn hash, Block hash, or Block height"
+              aria-label="Search"
+              aria-describedby="search"
+              onChange={this.handleSearchValueChange}
+              className="search-field"
+            />
+            {this.props.dashboard && (
+              <Button type="submit" variant="info" className="button-search">
                 Search
-              </span>
-            ) : (
-              <img
-                src="/static/images/icon-search.svg"
-                className="search-icon"
-              />
+              </Button>
             )}
-          </Col>
+          </InputGroup>
         </Row>
-      </Col>
-      <style jsx global>{`
-        .search {
-          border-radius: 25px;
-          border: solid 2px #e6e6e6;
-          background-color: #f8f8f8;
-          transition: all 0.3s;
-          overflow: hidden;
-        }
+        <style jsx global>{`
+          .search-box {
+            background: white;
+            width: ${this.props.dashboard ? "740px" : "500px"};
+            max-width: 100%;
+            height: ${this.props.dashboard ? "49px" : "40px"};
+            margin: auto;
+          }
+
+          @media (max-width: 1000px) {
+            .search-box {
+              width: 100%;
+            }
+          }
 
-        .search-icon-border {
-          border-left: solid 1px #e6e6e6;
-          cursor: pointer;
-          background: rgba(0, 0, 0, 0.1);
-          padding-top: 4px;
-          padding-bottom: 4px;
-        }
+          .input-group-text {
+            background: #fafafa;
+            border: 2px solid #eaebeb;
+            border-right: none;
+            border-radius: 8px;
+          }
 
-        .search-text {
-          font-family: BentonSans;
-          font-size: 14px;
-          color: #999999;
-          outline: none;
-          border: 1px solid #fff;
-          background-color: #f8f8f8;
-          font-weight: 100;
-          padding-top: 5px;
-          padding-bottom: 5px;
-        }
+          .search-field {
+            background: ${this.props.dashboard ? "#FFFFFF" : "#FAFAFA"};
+            border: 2px solid #eaebeb;
+            border-left: ${this.props.dashboard ? "2px solid #eaebeb" : "none"};
+            box-sizing: border-box;
+            border-radius: 8px;
+            height: 100%;
+            font-weight: lighter;
```

Comment:
Please, remove this rule since it is too light, and it is really hard to read. The default font-weight matches the Figma design.